src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxExtractOperation.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.SqlServer.Dac;
+using Microsoft.SqlTools.ServiceLayer.DacFx.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.Utility;
+using System;
+using System.Data.SqlClient;
+using System.Diagnostics;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx
+{
+    /// <summary>
+    /// Class to represent an in-progress extract operation
+    /// </summary>
+    class DacFxExtractOperation : DacFxOperation
```

Comment:
Seems like we're repeating a log of operation code - should we centralize this using a helper or a generic method?

Reply:
Moved the common code to base DacFxOperation class